Xwt/Xwt/WindowFrame.cs

```diff
@@ -74,6 +74,7 @@ public class WindowFrame: XwtComponent, IAnimatable
 		bool pendingReallocation;
 		Image icon;
 		WindowFrame transientFor;
+		protected bool owned;
```

Comment:
this should be `true` by default and only be false for `NativeWindowFrame`

Reply:
Yes, sorry, I screwed up.